config/config.go

```diff
@@ -9,8 +9,7 @@ import (
 
 // Config is the search API handler config
 type Config struct {
-	AwsRegion                  string        `envconfig:"AWS_REGION"`
-	AwsService                 string        `envconfig:"AWS_SERVICE"`
+	Aws                        AWS
```

Comment:
(suggestion): would be better to rename this to ```AWS``` for example AWSRegion, AWSService etc.